packages/sn-filter-pane/src/components/FoldedListbox/FoldedListbox.tsx

```diff
@@ -42,11 +42,69 @@ export const FoldedListbox = ({ resource, onClick, stores }: FoldedListboxProps)
     backgroundColor: stardustTheme?.getStyle('object', '', 'listBox.backgroundColor') ?? '#FFFFFF',
     color: stardustTheme?.getStyle('object', '', 'listBox.title.main.color') ?? '#404040',
     width: containerRef?.current?.clientWidth,
-  }));
+    '&:focus:not(:hover)': {
+      boxShadow: 'inset 0 0 0 2px #3F8AB3 !important',
+    },
+    '&:focus-visible': {
+      outline: 'none',
+    },
+  }),
+);
+
+export const FoldedListbox = ({ resource, onClick, stores }: FoldedListboxProps) => {
+  const fieldName = useFieldName(resource.layout);
+  const containerRef = useRef<HTMLDivElement>(null);
+  const {
+    constraints, stardustTheme, options, translator,
+  } = stores.store.getState();
+  const isRtl = options.direction === 'rtl';
+  const isDrillDown = resource.layout.qListObject.qDimensionInfo.qGrouping === 'H';
+
+  const changeFocus = (event: React.KeyboardEvent) => {
+    // @ts-ignore
+    event.target?.setAttribute('tabIndex', '-1');
+    // @ts-ignore
+    event.target?.blur();
```

Comment:
```suggestion
    const target = event.target as HTMLElement | undefined;
    target?.setAttribute('tabIndex', '-1');
    target?.blur();
```
(should fix type errors)

Reply:
Updated!